docs/basics/register_access.md

```diff
@@ -0,0 +1,401 @@
+    | X10       | Integer          | W10              | Temporary register                               |
+    | X11       | Integer          | W11              | Temporary register                               |
+    | X12       | Integer          | W12              | Temporary register                               |
+    | X13       | Integer          | W13              | Temporary register                               |
+    | X14       | Integer          | W14              | Temporary register                               |
+    | X15       | Integer          | W15              | Temporary register (also called "IP1")           |
+    | X16       | Integer          | W16              | Platform Register (often used as scratch)        |
+    | X17       | Integer          | W17              | Platform Register (often used as scratch)        |
+    | X18       | Integer          | W18              | Platform Register                                |
+    | X19       | Integer          | W19              | Callee-saved register                            |
+    | X20       | Integer          | W20              | Callee-saved register                            |
+    | X21       | Integer          | W21              | Callee-saved register                            |
+    | X22       | Integer          | W22              | Callee-saved register                            |
+    | X23       | Integer          | W23              | Callee-saved register                            |
+    | X24       | Integer          | W24              | Callee-saved register                            |
+    | X25       | Integer          | W25              | Callee-saved register                            |
+    | X26       | Integer          | W26              | Callee-saved register                            |
+    | X27       | Integer          | W27              | Callee-saved register                            |
+    | X28       | Integer          | W28              | Callee-saved register                            |
+    | X29       | Integer          | W29, FP          | Frame pointer                                    |
+    | X30       | Integer          | W30, LR          | Link register (return address)                   |
+    | XZR       | Integer          | WZR, ZR          | Zero register (always reads as zero)             |
+    | SP        | Integer          |                  | Stack pointer                                    |
+    | PC        | Integer          |                  | Program counter                                  |
+    | **Flags** |
+    | PSTATE      | Integer          |                  | [Processor state in exception handling](https://developer.arm.com/documentation/100933/0100/Processor-state-in-exception-handling)            |
+    | **Floating Point Registers (SIMD/FP)** |
+    | V0        | Integer   |                  | Vector or scalar register                        |
```

Comment:
This is valid also for the other archs. Do we really need to list all the vector registers that are different just for the number in the name? Can we do something line V0-V31 or V[0-31], or whatever?

Reply:
Initially I thought of contracting them, but then we would miss the aliases for some of them. Do we make it asymmetric?

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)